CMakeLists.txt

```diff
@@ -18,63 +18,71 @@ cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
 
 # Overrides must go before the project() statement, otherwise they are ignored.
 
-################################################################################
+# ##############################################################################
```

Comment:
Is that space intentional? Looks like the hashruler options affect this. Don't mind either way, just curious if it was on purpose.

Reply:
Yes, cmake-format inserts a space after the first `'#'`, no idea why. I'd disable this, but then the tools stops normalizing the length of hash rulers. In the end, I'm completely impartial to this. Let me know how you like it most.

Reply:
If the space means the lengths are consistent let's go with that, I don't mind.